src/service/extensions-impl.js

```diff
@@ -440,8 +440,9 @@ export class Extensions {
     scriptElement.setAttribute('custom-element', extensionId);
     scriptElement.setAttribute('data-script', extensionId);
     const pathStr = this.win.location.pathname;
+    const base = this.win.location.protocol + '//' + this.win.location.host;
```

Comment:
We'll need port number, too. How about passing in `window.location`, and let calculateExtensionScriptUrl figure out what it needs.